quokka/core/tests/test_models.py

```diff
@@ -1,8 +1,13 @@
 # coding: utf-8
+import sys
+
 from . import BaseTestCase
 
 from ..models import Channel, Config, CustomValue
 
+if sys.version_info.major == 3:
+    unicode = lambda x: u'{}'.format(x)
```

Comment:
On 17.05.2015 17:22, Diego Garcia wrote:
>
> In quokka/core/tests/test_models.py
> <https://github.com/quokkaproject/quokka/pull/190#discussion_r30471298>:
>
> >  from . import BaseTestCase
> >  
> >  from ..models import Channel, Config, CustomValue
> >  
> > +if sys.version_info.major == 3:
> > +    unicode = lambda x: u'{}'.format(x)
>
> Awesome :smile:
>
>
Umm, why is that any different from "unicode = str" ?

-- 
-- Matthias Urlichs